tests/zfs-tests/tests/functional/alloc_class/alloc_class_013_pos.ksh

```diff
@@ -44,7 +44,7 @@ log_must zfs create -o dedup=on -V 2G $TESTPOOL/$TESTVOL
 
 log_must eval "new_fs $ZVOL_DEVDIR/$TESTPOOL/$TESTVOL >/dev/null 2>&1"
 
-sync_pool
+log_must sync
```

Comment:
```suggestion
sync_pool $TESTPOOL
```
?

Reply:
Yeah, not sure how the wires got crossed there.